### ISSUE TITLE
train images with different width and height sizes?

### ISSUE THREAD

**Issue by SteveIb** [jfzhang95/pytorch-deeplab-xception#171]
Hi,

Can I train images with different width and height sizes? like 1024*512, since my image scale is 2:1

the default size is 513 in this repo, how can I change it if possible?

**Comment by dragen1860** [comment]
@SteveIb Yes, segmentation task will usually resize images while keeping image aspect. and then the image will be padded to same size to be put into a batch.

**Comment by SteveIb** [comment]
Hi @dragen1860 how it is gonna keep the image aspect while it is resized to 513 by 513?
